src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -960,8 +993,8 @@ internal void UnregisterSystemTarget(SystemTarget target)
         public string GetDebugDump(bool all = true)
         {
             var sb = new StringBuilder();            
-            foreach (var sytemTarget in activationDirectory.AllSystemTargets())
-                sb.AppendFormat("System target {0}:", sytemTarget.GrainId.ToString()).AppendLine();               
+            foreach (var systemTarget in activationDirectory.AllSystemTargets())
+                sb.AppendFormat("System target {0}:", ((ISystemTargetBase)systemTarget).GrainId.ToString()).AppendLine();               
```

Comment:
No, it's necessary because `GrainId` is on internal `ISystemTargetBase`, not on public `ISystemTarget`. The naming is a bit misleading I guess.